backend/corpora/parliament/utils/parlamint_v4.py

```diff
@@ -98,8 +98,8 @@ def extract_person_data(node):
         'name': name,
         'role': role,
         'gender': gender,
-        'party_ids': party_ids,
-        'party_nodes': party_nodes,
+        'org_ids': org_ids,
+        'org_nodes': org_nodes,  # unsure about this still: might be a bit of a memory leak to store the nodes in the metadata
```

Comment:
By which I mean: Nodes are pretty big in BeautifulSoup, so to store them all for each person seems not ideal. I can look at this later myself too, just haven't gotten around to it yet. The problem is just that `org_nodes` is currently used to extract the role in the specific corpus definition...

Reply:
The word is "expensive" or perhaps "inefficient". "Memory leak" means something more specific, which is also more problematic but which is fortunately not what you mean here. A memory leak is always expensive but the opposite is not true.

(I recommend changing the wording because a memory leak is often a serious bug that must be hunted down. No need to put out a red flag here.)